deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/security/TokenAuthFilter.java

```diff
@@ -52,9 +52,13 @@ public TokenAuthFilter(String userDataUrl, String groupDataUrl, String userNameK
     public void filter(ContainerRequestContext context) throws IOException {
         if(!context.getMethod().equals("OPTIONS")) {
             SecurityContext securityContext;
-
             try {
                 securityContext = authenticate(context);
+                if (securityContext == null ){
+                    throw new TeletaanInternalException(
+                        Response.Status.UNAUTHORIZED, "Unrecognized user credential");
+                }
```

Comment:
`authenticate` below returns null when it's healthcheck. I think we only need to log when it's not null, and do nothing if it's null.